addons/mail/static/src/js/thread_windows/thread_window.js

```diff
@@ -59,19 +61,27 @@ var ThreadWindow = AbstractThreadWindow.extend({
         if (!this.hasThread()) {
             this._startWithoutThread();
         } else if (this.needsComposer()) {
```

Comment:
@alexkuhn  the function needsComposer was removed in your commit, does it still work ?